resources/js/core.js

```diff
@@ -192,12 +194,11 @@ const photoBooth = (function () {
 
                 $('.spinner').hide();
                 $('.loading').empty();
-                $('<p>').text(`${result.current + 1} / ${result.limit}`).appendTo('.loading');
 
                 if (config.continuous_collage) {
                     setTimeout(() => {
                         public.thrill('collage');
-                    }, 1000);
+                    }, 0);
```

Comment:
This is completely independent from your pr and the better solution would be to make this configurable.

Reply:
I reverted this.